rai/test_utils.go

```diff
@@ -0,0 +1,111 @@
+// Copyright 2022 RelationalAI, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package rai
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func NewTestClient() (*Client, error) {
+	configPath, _ := expandUser(DefaultConfigFile)
+	if _, err := os.Stat(configPath); err == nil {
+		return NewDefaultClient()
+	}
+
+	var cfg Config
+
+	clientId := os.Getenv("CLIENT_ID")
+	clientSecret := os.Getenv("CLIENT_SECRET")
+	clientCredentialsUrl := os.Getenv("CLIENT_CREDENTIALS_URL")
+
+	placeHolderConfig := `
```

Comment:
why is it a "placeholder"? this is a format string.

Reply:
I think we changed the content but not the name, renaming this to `configFmt` :pray: